common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -155,4 +155,14 @@ trait ABTestSwitches {
     sellByDate = new LocalDate(2018, 1, 10),
     exposeClientSide = true
   )
+
+  Switch(
+    ABTests,
+    "ab-acquisitions-banner-just-contribute",
+    "Test linking to a contributions-only version of support.theguardian.com",
+    owners = Seq(Owner.withGithub("joelochlann")),
+    safeState = Off,
+    sellByDate = new LocalDate(2018, 1, 20),
```

Comment:
Your tests are failing because this is a Saturday, and tests switches aren't allowed to expire at the weekend